index.html

```diff
@@ -39,5 +40,144 @@ <h2>LET'S CONNECT</h2>
+                <ol>
+                    <li>CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                </ol>
+
+                <p>A daily selection of privately<br>
+                personalised reads: no accounts or <br>
+                sign-ups required.</p><br>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascrpt</li>
+                </ul>
+                   <button type="button" class="button">See project</button>
+            </div>
+            </section>
+            <section class="about">
+                 <h2>About<br>Myself</h2>
+                <p> Hello i'm a software developer! I can help<br>
+                you build a product, feature or website<br>
+                look through some of my work and<br>
+                experience! if you like what you see and <br>
+                have a project you need coded, don't<br>
+                hestiate to contact me.</p>
+
+                <h3>LET'S CONNECT</h3>
+
+                <ul class="icons">
+                    <li>
+                        <a href="#"><img src="tweet.svg" alt="icons"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="Linkedinicon.png" alt="icons"></a>
+                    </li>            <li>
+                        <a href="#"><img src="Vector1.png" alt="icons"></a>
+                    </li>            <li>
+                        <a href="#"><img src="Vector2.png" alt="icons"></a>
+                    </li>            <li>
+                        <a href="#"><img src="Vector3.png" alt="icons"></a></li>
+                    </ul> 
+
+                    <button type="button" class="button">Get My Resume</button>
+                </section>
+
+                    <ul class="skills">
+                         <li>
+                            <h2>Languages <i class="fas fa-angle-down"></i></h2>
+                         </li>
+                         </ul>
+                            <ul class="tech-skills">
+                               <li>
+                                  <div class="tech-skills-detail">
+                                    <img src="Ellipse1(2).png" alt="Javascript">
+                                    <p>Javascript</p>
+                                 </div>
+                               </li>
+                               </ul>
+                               <ul class="tech">
+                               <li>
+                              <div class="tech-skills-detail">
+                                 <img src="Ellipse1(3).png" alt="HTML">
+                                 <p>HTML</p>
+                               </div>
+                            </li>
+                       
+                           <li>
+                             <div class="tech-skills-detail">
```

Comment:
Your about section doesn't look like the one in Figma design ,I encourage you to try to stick to Figma design as much as possible 

- [ ]  try to remove the space from left 

- [ ] try to make the button as the one in the Figma design 


- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/166077757-f0f85d52-0cea-4738-99ee-7e272f8d0b34.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/166077849-4550c0cb-96fe-47d5-aa75-f73208fd2125.png)